kokoro/windows/build.bat

```diff
@@ -89,13 +89,15 @@ ninja check-copyright
 if %ERRORLEVEL% GEQ 1 exit /b %ERRORLEVEL%
 echo "Build Completed %DATE% %TIME%"
 
-:: #########################################
-:: Run the tests.
-:: #########################################
+:: ################################################
+:: Run the tests (We no longer run tests on VS2013)
+:: ################################################
+if NOT %VS_VERSION% == 2013 (
```

Comment:
So we're still going to build 2013 but not test it?  Why not wipe out 2013 completely, say it's not supported, delete the .bat and .cfg files?   Why leave an untested configuration hanging around?

Reply:
Yes, would love to do that. 
We've had this discussion with others in the ecosystem, and have pushed for what you suggest.  We get strong pushback.  I won't rehash the arguments here.

This is the compromise.